Maddori.Apple/Maddori.Apple/Screen/Home/Home/UIComponent/ChangeTeamView.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  ChangeTeamView.swift
+//  Maddori.Apple
+//
+//  Created by 이성호 on 2023/02/06.
+//
+
+import UIKit
+
+import SnapKit
+
+final class ChangeTeamView: UIView {
+    
+    // MARK: - FIXME: 데이터 더미 입니다.
+    let teamDataDummy: [String] = ["맛쟁이 사과처럼", "명예 소방관"]
+    
+    // MARK: - property
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = .label2
+        label.text = TextLiteral.changeTeamViewLabel
+        label.textColor = .black100
+        return label
+    }()
+    private let collectionViewFlowLayout: UICollectionViewFlowLayout = {
+        let flowLayout = UICollectionViewFlowLayout()
+        flowLayout.scrollDirection = .vertical
+        flowLayout.sectionInset = UIEdgeInsets(top: 12, left: 0 , bottom: 16, right: 0)
+        flowLayout.itemSize = CGSize(width: UIScreen.main.bounds.width - SizeLiteral.leadingTrailingPadding*2, height: 59)
```

Comment:
p3;

```suggestion
        flowLayout.itemSize = CGSize(width: UIScreen.main.bounds.width - SizeLiteral.leadingTrailingPadding * 2, height: 59)
```

앞 뒤 한칸씩 부탁드립니다!

Reply:
[58fb548](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/319/commits/58fb548dcbf1ccc9f3aa0b5376f899fca9d4b153)

수정했습니다.